src/xcode/ENA/ENA/Source/Extensions/URLSession+Default.swift

```diff
@@ -10,8 +10,7 @@ extension URLSession {
 		delegateQueue: OperationQueue? = nil
 	) -> URLSession {
 		#if DISABLE_CERTIFICATE_PINNING
-		/// Disable certificate pinning while app is running on:
-		/// Community, Debug, TestFlight, UITesting modes
+		/// Disable certificate pinning with build condition for manual debugging if set explicitly.
```

Comment:
But this is not set manual, it is set in the build settings for the different targets.

Reply:
I see that its set on `Community` and `Debug` configuration but not on `TestFlight`

Reply:
Ah okay, thanks. So I've adjusted the comment. :slightly_smiling_face: